### ISSUE TITLE
Remove POMS clause in jobsub_lite version of condor_vault_storer script since POMS will no longer support analysis

### ISSUE THREAD

**Issue by shreyb** [fermitools/jobsub_lite#571]

**Comment by marcmengel** [comment]
Note that if we're doing that ,we can probably drop having a jobsub-specific condor_vault_storer altogether...

**Comment by shreyb** [comment]
Agreed.  That's the plan.  We could slate it for this release, or we could wait until the CILogon proxy shutoff transition dies down and then do this.  What do you think?

**Comment by marcmengel** [comment]
Hm... I haven't heard any sort of announcement about dropping Analysis stuff from POMS.   And it sounds like Mu2e are about to request it continue to  be supported.  On the other hand, I think we have reached the point that the changes we need are upstreamed assuming we set the right environment variables for the stock condor_vault_storer...

**Comment by shreyb** [comment]
Marc just checked production and dev POMS.  We should be good to test by simply commenting out the line in `lib/condor.py` where we set `_condor_SEC_CREDENTIAL_STORER`, and test.  If all looks good, we'll remove that bit and delete our version of `condor_vault_storer` from `bin/`.

**Comment by shreyb** [comment]
@marcmengel, @vitodb  - is this still relevant?  I believe POMS handles everything it needs wrt to `condor_vault_storer`, so do we need the jobsub_lite version anymore?

**Comment by marcmengel** [comment]
I'm pretty sure all our changes got integrated in the system condor_vault_storer.  We should of course test to verify;esp. on the POMS
development machine. to make sure POMS Analysis jobs launch okay.
